proposal/signatories.Rmd

```diff
@@ -15,7 +15,7 @@ Who are the people responsible for actually delivering the project if the propos
 -->
 ```
 -   [Elio Campitelli](https://eliocamp.github.io/): Ph.D. student at the University of Buenos Aires in atmospheric sciences and an R package developer.
-    They are a founding member of the R User Group in Buenos Aires and MetaDocencia.
+    They are a founding member of the R User Group in Buenos Aires, and of MetaDocencia.
```

Comment:
should there be a link for metadocencia by the way?

Reply:
I think those descriptions are a bit long and I'm not active in MetaDocencia any more, so I might just remove that part altogether.